### DISCUSSION TITLE
Strange error

### DISCUSSION THREAD

**Discussion by lucadelu** [davidfrantz/force#312] [Q&A]
Hi,

I'm using FORCE to process Sentinel and Landsat scenes, but sometimes I got this error

simplex2.c:372: ERROR: non-finite function value encountered
Default GSL error handler invoked. /home/docker/bin/force-l2ps_: line 161: 355
Aborted (core dumped)

do you have any idea why this happen?

thanks a lot
Luca

**Comment by davidfrantz** [comment]
Hi Luca,

does this happen with the provided Docker container?

Best,
David

**Reply by lucadelu** [reply]
Hi David,

no, it happen also with your, my problem with gdal was after running force.

Could depend somehow with DTM not covering entirely the scene?

**Comment by davidfrantz** [comment]
Potentially. Can you send me one of those image names (or even better a download path)? 

I will try to reproduce here.
